src/documents/Builder.md

```diff
@@ -103,6 +103,36 @@ var myBuilder = fluid.prefs.builder({
 ```
 It is not necessary to specify the primary schema; The builder will automatically find the preference specifications provided by the Framework and build a primary schema (see [Processing the Schemas](#processing-the-schemas) below for more information).
 
+#### Switching Between Native HTML and jQuery UI Widgets
```

Comment:
Was this intended to be a subsection of "Using the Starter Preferences"? I feel like it could be at the same level, but can be convinced either way. Also could you add the #s to the end of the header too, just to make it easier to see in the file.